sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/DocumentAnalysisAsyncClient.java

```diff
@@ -380,6 +384,189 @@ public final class DocumentAnalysisAsyncClient {
         }
     }
 
+    /**
+     * Classify a given document using a document classifier.
+     * For more information on how to build a custom classifier model,
+     * see <a href="https://aka.ms/azsdk/formrecognizer/buildclassifiermodel"></a>
+     * <p>The service does not support cancellation of the long running operation and returns with an
+     * error message indicating absence of cancellation support.</p>
+     *
+     * <p><strong>Code sample</strong></p>
+     * <p> Analyze a document using the URL of the document. </p>
+     * <!-- src_embed com.azure.ai.formrecognizer.documentanalysis.DocumentAnalysisAsyncClient.beginClassifyDocumentFromUrl#string-string -->
+     * <pre>
+     * String documentUrl = &quot;&#123;document_url&#125;&quot;;
+     * String modelId = &quot;&#123;model_id&#125;&quot;;
+     * documentAnalysisAsyncClient.beginAnalyzeDocumentFromUrl&#40;modelId, documentUrl&#41;
+     *     &#47;&#47; if polling operation completed, retrieve the final result.
+     *     .flatMap&#40;AsyncPollResponse::getFinalResult&#41;
+     *     .subscribe&#40;analyzeResult -&gt;
+     *         analyzeResult.getDocuments&#40;&#41;
+     *             .stream&#40;&#41;
+     *             .forEach&#40;document -&gt;
+     *                 document.getFields&#40;&#41;
+     *                     .forEach&#40;&#40;key, documentField&#41; -&gt; &#123;
+     *                         System.out.printf&#40;&quot;Field text: %s%n&quot;, key&#41;;
+     *                         System.out.printf&#40;&quot;Field value data content: %s%n&quot;, documentField.getContent&#40;&#41;&#41;;
+     *                         System.out.printf&#40;&quot;Confidence score: %.2f%n&quot;, documentField.getConfidence&#40;&#41;&#41;;
+     *                     &#125;&#41;&#41;&#41;;
```

Comment:
Sample looks like it's for beginAnalyzeDocument (shows fields) but `classifyDocument` doesn't return fields.